server/admin_management/api/app/routers/router_application_admin.py

```diff
@@ -75,34 +77,32 @@ def create_application_admin(
     )
 
     try:
-        audit_event_log.requesting_user = user_service.get_user_by_cognito_user_id(
-            requester.cognito_user_id
-        )
+        audit_event_log.requesting_user = requester
         audit_event_log.application = application_service.get_application(
             application_admin_request.application_id
         )
 
-        return application_admin_service.create_application_admin(
+        new_application_admin = application_admin_service.create_application_admin(
             application_admin_request, target_user, requester.cognito_user_id
         )
 
+        # get target user from database, so for existing user, we'll have get the cognito user id
```

Comment:
Sorry Catherine, do you mean "we can get"?

Reply:
yes, sorry hh